src/components/reception/MultiplexedLibraryBarcode.vue

```diff
@@ -0,0 +1,318 @@
+    </div>
+    <div id="print" class="p-2 bg-gray-100 rounded p-3">
+      <traction-heading level="4" :show-border="true">Print labels</traction-heading>
+      <div class="flex flex-row space-x-4 w-full">
+        <div class="flex flex-col w-1/2 text-left space-y-2">
+          <traction-label>Barcodes</traction-label>
+          <textarea
+            id="print-barcodes"
+            v-model="printBarcodes"
+            class="w-full py-2 px-3 border border-gray-300 bg-white rounded-md shadow-sm focus:outline-none focus:ring-sdb-100 focus:border-sdb-100 disabled:opacity-75 disabled:bg-gray-200 disabled:cursor-not-allowed"
+            rows="4"
+            disabled
+            max-rows="10"
+          />
+        </div>
+        <DataFetcher :fetcher="fetchPrinters">
+          <div class="flex flex-col text-left w-full space-y-2">
+            <traction-label>Printers</traction-label>
+            <traction-select
+              id="printer-choice"
+              v-model="printerName"
+              data-attribute="printer-options"
+              :options="printerOptions"
+              value-field="text"
+              :classes="'p-0'"
+              required
+            />
+          </div>
+        </DataFetcher>
+      </div>
+      <div class="flex justify-end mt-3 w-full">
+        <traction-button
+          id="print-button"
+          class="grow"
+          theme="print"
+          :disabled="!printEnabled"
+          @click="printLabels"
+          >Print Labels</traction-button
+        >
+      </div>
+    </div>
+    <div class="bg-gray-100 rounded p-3">
+      <traction-heading level="4" :show-border="true"> Summary </traction-heading>
+      <div class="flex flex-col w-full">
+        <traction-muted-text class="text-left">Imports data into Traction</traction-muted-text>
+        <p id="importText" class="text-left">
+          Import {{ labwareData.foundBarcodes.size }} labware into {{ props.pipeline }} from
+          {{ reception.text }}
+        </p>
+        <div class="flex flex-row space-x-8 mt-5">
+          <traction-button
+            id="reset"
+            full-width
+            theme="reset"
+            data-action="reset-form"
+            @click="reset"
+          >
+            Reset
+          </traction-button>
+          <traction-button
+            id="importLabware"
+            :disabled="isDisabled"
+            full-width
+            theme="create"
+            data-action="import-labware"
+            @click="importLabware"
+          >
+            <traction-spinner v-if="isFetching" class="h-5"></traction-spinner>
+            <span v-else class="button-text">Import</span>
+          </traction-button>
+        </div>
+      </div>
+    </div>
+  </DataFetcher>
+</template>
+
+<!-- 
+     A custom form for importing single multiplexed libraries into traction.
+     Used by lib/receptions/index.js when defining the method of importing barcodes in the reception.  
+  -->
```

Comment:
Yes there probably is but I think it might need to be saved for another time as this PR is getting quite large and complex.
The benefit of having them completely separated is they have no dependencies on one another but there is some code duplication that could probably be cleaned up.

Reply:
Yes, I agree. The way you have currently designed is already good which allows isolation and flexibility to future extensions using configuration object.